R/PackageDependencyReporter.R

```diff
@@ -26,9 +26,6 @@ DependencyReporter <- R6::R6Class(
     "DependencyReporter",
     inherit = AbstractGraphReporter,
 
-    #TODO [patrick.boueri@uptake.com]: Add more robust error checks and logging
```

Comment:
this is legacy from an early version of `pkgnet`, the first time @Paxanator was like "hey classes are cool you should use classes".